packages/@aws-cdk/aws-lambda-event-sources/package.json

```diff
@@ -109,7 +109,13 @@
       "docs-public-apis:@aws-cdk/aws-lambda-event-sources.KinesisEventSourceProps",
       "docs-public-apis:@aws-cdk/aws-lambda-event-sources.S3EventSourceProps",
       "props-default-doc:@aws-cdk/aws-lambda-event-sources.S3EventSourceProps.filters",
-      "docs-public-apis:@aws-cdk/aws-lambda-event-sources.SqsEventSourceProps"
+      "docs-public-apis:@aws-cdk/aws-lambda-event-sources.SqsEventSourceProps",
+      "duration-prop-type:@aws-cdk/aws-lambda-event-sources.DynamoEventSourceProps.maximumRecordAgeInSeconds",
+      "duration-prop-type:@aws-cdk/aws-lambda-event-sources.KinesisEventSourceProps.maximumRecordAgeInSeconds",
+      "duration-prop-name:@aws-cdk/aws-lambda-event-sources.KinesisEventSourceProps.maximumRecordAgeInSeconds",
+      "duration-prop-type:@aws-cdk/aws-lambda-event-sources.StreamEventSourceProps.maximumRecordAgeInSeconds",
+      "duration-prop-name:@aws-cdk/aws-lambda-event-sources.StreamEventSourceProps.maximumRecordAgeInSeconds",
+      "duration-prop-name:@aws-cdk/aws-lambda-event-sources.DynamoEventSourceProps.maximumRecordAgeInSeconds"
```

Comment:
We should need none of these exceptions. I believe addressing my previous comment on using `Duration` should fix this. If not, run `yarn awslint list` to see what each of these checks are and see how they can be resolved.

Reply:
This will 100% be fixed by addressing that previous comment.

Reply:
This is cleaned up now